eZ/Publish/Core/Repository/UserService.php

```diff
@@ -1135,6 +1160,9 @@ protected function createPasswordHash($login, $password, $site, $type)
             case User::PASSWORD_HASH_PLAINTEXT:
                 return $password;
 
+            case User::PASSWORD_BCRYPT:
+                return password_hash($password, PASSWORD_BCRYPT);
```

Comment:
The doc says PASSWORD_BCRYPT will always return a 60 char string, so why set the DB column to 255?